### PR TITLE
Url update

### PR DESCRIPTION
**Pull request by yahya-cloud**

### CODE REVIEW

File: docs/theming_plone_5/custom-components.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Plone has default Viewlets and ViewletManagers like ContentAbove and BelowConten
 For an overview of existing Viewlets and ViewletManagers look at the `/@@manage-viewlets` view.
 
 For more details about Viewlets/ViewletManagers and there possibilities see the Viewlets sections of the
-[Mastering Plone 5: Development chapters](https://training.plone.org/5/mastering-plone/viewlets_1.html).
+[Mastering Plone 5: Development chapters](https://2022.training.plone.org/mastering-plone/viewlets_1.html).
```
Comment (stevepiercy) [inline docs/theming_plone_5/custom-components.md:39]: Not being archived. See: https://docs.google.com/spreadsheets/d/1Wmg03uXD3fIRYz33xPFnzaFbLYdxjhKq10NAKDswFxg/edit#gid=0

```suggestion
[Mastering Plone 5: Development chapters](https://training.plone.org/mastering-plone/viewlets_1.html).
```

Comment (yahya-cloud) [reply docs/theming_plone_5/custom-components.md]: I'm sorry about that, i updated it now

File: docs/mastering-plone-5/code.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ git checkout views_2
 ```
 
 The names of the tags are the same as the URL of the chapter.
-The tag for the chapter <https://training.plone.org/5/mastering-plone/registry.html> is `registry`
+The tag for the chapter <https://training.plone.org/mastering-plone/registry.html> is `registry`
```
Comment (stevepiercy) [inline docs/mastering-plone-5/code.md:36]:
```suggestion
The tag for the chapter https://training.plone.org/mastering-plone/registry.html is `registry`.
```

File: docs/mastering-plone-5/code.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ git checkout views_2
...
 and you can get it with {command}`git checkout registry`.
```
Comment (stevepiercy) [inline docs/mastering-plone-5/code.md:37]:
```suggestion
You can get it with {command}`git checkout registry`.
```

File: docs/theming_plone_5/custom-components.md [hunk 4]
```diff
@@ -36,7 +36,7 @@ Plone has default Viewlets and ViewletManagers like ContentAbove and BelowConten
 For an overview of existing Viewlets and ViewletManagers look at the `/@@manage-viewlets` view.
 
 For more details about Viewlets/ViewletManagers and there possibilities see the Viewlets sections of the
-[Mastering Plone 5: Development chapters](https://training.plone.org/5/mastering-plone/viewlets_1.html).
+[Mastering Plone 5: Development chapters](https://training.plone.org/mastering-plone/viewlets_1.html).
```
Comment (stevepiercy) [inline docs/theming_plone_5/custom-components.md:39]:
```suggestion
```

File: docs/theming_plone_5/custom-components.md [hunk 5]
```diff
@@ -36,7 +36,7 @@ Plone has default Viewlets and ViewletManagers like ContentAbove and BelowConten
 For an overview of existing Viewlets and ViewletManagers look at the `/@@manage-viewlets` view.
 
 For more details about Viewlets/ViewletManagers and there possibilities see the Viewlets sections of the
```
Comment (stevepiercy) [inline docs/theming_plone_5/custom-components.md:38]: Let's use MyST syntax, and combine two lines into one.

```suggestion
For more details about Viewlets/ViewletManagers and their possibilities, see {doc}`mastering-plone/viewlets_1`.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
While we are cleaning up links, let's use proper MyST syntax.

Also correct spelling and simplify a bit.

**Comment by yahya-cloud** [comment]
thank you for your help, i have made he requested changes now can you check if its correct

File: docs/theming_plone_5/custom-components.md [hunk 6]
```diff
@@ -35,8 +35,7 @@ Plone has default Viewlets and ViewletManagers like ContentAbove and BelowConten
 
 For an overview of existing Viewlets and ViewletManagers look at the `/@@manage-viewlets` view.
 
-For more details about Viewlets/ViewletManagers and there possibilities see the Viewlets sections of the
-[Mastering Plone 5: Development chapters](https://training.plone.org/5/mastering-plone/viewlets_1.html).
+For more details about Viewlets/ViewletManagers and their possibilities, see {doc}`mastering-plone/viewlets_1`.
```
Comment (stevepiercy) [inline docs/theming_plone_5/custom-components.md:38]: My mistake, I forgot to make the path root-relative. However can you check this by making a local build? That will save review cycles. If not, then you can always check the Netlify preview build. For example, the latest build shows that this link did not render: https://deploy-preview-742--plone-training.netlify.app/theming_plone_5/custom-components#viewlets

```suggestion
For more details about Viewlets/ViewletManagers and their possibilities, see {doc}`/mastering-plone/viewlets_1`.
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: docs/theming_plone_5/custom-components.md (hunk 6 above)
Comment (yahya-cloud) [reply docs/theming_plone_5/custom-components.md]: I'll definitely keep this mind from next time, now i have updated the file and the link do renders in Netlify preview build

**Review by stevepiercy — APPROVED** [review]
Thank you for your patience and flexibility with all my change requests. I appreciate it. LGTM!

**Pull request merged**